app/components/createService/createservice.html

```diff
@@ -445,6 +446,52 @@
           <!-- tab-placement -->
           <div class="tab-pane" id="placement" ng-include="'app/components/createService/includes/placement.html'"></div>
           <!-- !tab-placement -->
+          <!-- tab-resources -->
+          <div class="tab-pane" id="resources">
+            <form class="form-horizontal" style="margin-top: 15px;">
+              <div class="form-group">
```

Comment:
Could you place the CPU/MEM res block on the left and the CPU/MEM limit on the right ?